modules/virt-example-vf-host-services.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+[id="virt-example-vf-host-services_{context}"]
+= Example: Node network configuration policy for virtual functions 
+
+Update host network settings for SR-IOV network virtual functions (VF) in an existing cluster by applying a `NodeNetworkConfigurationPolicy` manifest.
+
+You can apply a `NodeNetworkConfigurationPolicy` manifest to an existing cluster to complete tasks such as the following:
+
+* Configure QoS, MTU, or miimon host network settings for VFs to optimize performance.
+* Add, remove or update VFs for a network interface.
+* Manage VF bonding configurations. 
+
+[NOTE]
+====
+To update host settings for SR-IOV network VFs by using NMState in clusters that also have the SR-IOV Network Operator installed, you must set the `externallyManaged` specification to `True` in the `SriovNetworkNodePolicy` resource. For more information, see the _Additional resources_ section.
+====
+
+The following YAML file is an example of a manifest that defines QoS policies for a VF.
+It includes samples values that you must replace with your own information.
+
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: qos <1>
+spec:
+  nodeSelector: <2>
+    node-role.kubernetes.io/worker: "" <3>
+  desiredState:
+    interfaces:
+      - name: ens1f0 <4>
+        description: Change QOS on VF0 <5>
+        type: ethernet <6>
+        state: up <7>
+        ethernet:
+         sr-iov:
+           total-vfs: 3 <8>
+           vfs:
+           - id: 0 <9>
+             max-tx-rate: 200 <10>
+----
+
+<1> Name of the policy.
+<2> Optional: If you do not include the `nodeSelector` parameter, the policy applies to all nodes in the cluster.
+<3> This example applies to all nodes with the `worker` role.
+<4> Name of the physical function (PF) network interface.
+<5> Optional: Human-readable description of the interface.
+<6> The type of interface.
+<7> The requested state for the interface after configuration.
+<8> The total number of VFs.
+<9> Identifies the VF with an ID of `0`.
+<10> Sets a maximum transmission rate, in Mbps, for the VF. This sample value sets a rate of 200 Mbps.
+
+The following YAML file is an example of a manifest that changes the bond mode configuration on a VF.
```

Comment:
No VF is involved in this example. Consider moving this into its own example section.

Reply:
I have commented out the example for the moment.